.github/workflows/workflow.yml

```diff
@@ -84,68 +88,91 @@ jobs:
       fail-fast: false
     steps:
       - name: Runtime environment
-        shell: bash
         env:
           WORKSPACE: ${{ github.workspace }}
         run: |
           echo "$HOME/.local/bin" >> $GITHUB_PATH
           echo "GITHUB_WORKSPACE=`pwd`" >> $GITHUB_ENV
+      - name: Coverage environment
```

Comment:
It would be good, if possible, to do a build without coverage and a build with so both paths in the build system are operated and exercised and so we can be confident we don't have any "works in debug mode" vs "works in release mode" issues.

This might well be achieved by putting this block after the install block below and having it blow away the build directory, re-running the configure, and build steps after this

(Same question for the other OS blocks below)